hazelcast/src/main/java/com/hazelcast/transaction/impl/TransactionContextImpl.java

```diff
@@ -129,6 +132,18 @@ public TransactionalObject getTransactionalObject(String serviceName, String nam
         return obj;
     }
 
+    private boolean requiresBackupLogs(String serviceName) {
```

Comment:
If the developer would miss it, then it would not lead to a bad thing; the backup logs are created. That is why I went for the 'on by default unless explicitly disabled'. 

But I can change it to 'off by default'